lib/rubycritic/commands/status_reporter.rb

```diff
@@ -0,0 +1,45 @@
+module Rubycritic
+  module Command
+    class StatusReporter
+      attr_reader :status, :status_message
+      SUCCESS = 0
+      SCORE_BELOW_MINIMUM = 1
+
+      def initialize(options)
+        @options = options
+        @status = SUCCESS
+      end
+
+      def score=(score)
+        return if score.nil?
+
+        @score = score.to_f
+        update_status
+        update_status_message
```

Comment:
That's confusing since it's not obvious what the difference between those 2 methods.
Additionally, you're decoupling something that **must** be coupled, right? There's no point in updating the status without updating the message?
I'd rather merge "update_status_message" with "update_status"